aflr/api_resources/voice.py

```diff
@@ -10,5 +10,5 @@ def __init__(self):
         self.url = self.api_base + "/voice"
 
     # get voices list
-    def list(self):
-        return self._get_request(url=self.url)
+    def list(self, **params):
```

Comment:
Do we have a list of the params somewhere in the SDK? Or just in the docs? If so maybe a link to the docs would be useful.

Reply:
its in the docs of the SDK.
Will add dataclasses in the near future.